swift/Samples/SplitScreenContainer/Sources/SplitScreenContainerScreen.swift

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import WorkflowUI
+
+
+/// A `SplitScreenContainerScreen` displays two screens side by side with a separator in between.
+public struct SplitScreenContainerScreen: Screen {
+
+    /// The screen displayed to the left of the separator.
+    public var leftScreen: AnyScreen
+
+    /// The screen displayed to the right of the separator.
+    public var rightScreen: AnyScreen
```

Comment:
Can we make this generic?

Reply:
Resolved via https://github.com/square/workflow/pull/925/commits/8c1c87ac1b8416541eb4b40597ddba5561f20eb0